<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindOpDlg</class>
 <widget class="QDialog" name="FindOpDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find operators</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>What:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_modeCb">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Operators by user name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operators by name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operators by category</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operators by type (&quot;category :: name&quot;)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>References to operator with user name</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="m_whatEdit"/>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="m_findBtn">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="m_caseSensitiveCb">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_opTree">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>120</number>
     </attribute>
     <column>
      <property name="text">
       <string>Page</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Position (x/y)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_gotoBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Goto</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelBtn">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_whatEdit</tabstop>
  <tabstop>m_caseSensitiveCb</tabstop>
  <tabstop>m_findBtn</tabstop>
  <tabstop>m_opTree</tabstop>
  <tabstop>m_gotoBtn</tabstop>
  <tabstop>m_cancelBtn</tabstop>
  <tabstop>m_modeCb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
